docs/reference/commandline/run.md

```diff
@@ -144,6 +144,8 @@ Options:
   -w, --workdir string              Working directory inside the container
 ```
 
+## Descriptino
```

Comment:
Oh, there's a PR somewhere that fixes this, but otherwise just add a commit :smile: